test/spicepods/tpch/sf1/federated/databricks[sql_warehouse].yaml

```
@@ -1,16 +1,16 @@
 version: v1
 kind: Spicepod
-name: databricks[federated]-sql_warehouse
+name: databricks[sql_warehouse]-federated
```

```suggestion
name: databricks[federated]-sql_warehouse
```

@peasee and I set the convention. The "trait" or "version" of it is last.

---

That isn't the convention we're using in the other files, i.e its currently databricks[delta_lake]-federated

---

https://github.com/spiceai/spiceai/blob/trunk/test%2Fspicepods%2FREADME.md

---

The other files are wrong :-)

---

Ah, nevermind, I'm wrong :-)

---

The variant here is the databricks mode, the federated part is to denote it's not accelerated - which matches the written convention